### PR TITLE
guile: cleanup formula after 3.0.9

### PR DESCRIPTION
**Pull request by aconchillo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/guile.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Guile < Formula
   mirror "https://ftpmirror.gnu.org/guile/guile-3.0.9.tar.xz"
   sha256 "1a2625ac72b2366e95792f3fe758fd2df775b4044a90a4a9787326e66c0d750d"
   license "LGPL-3.0-or-later"
+  revision 1
```
Comment (SMillerDev) [inline Formula/guile.rb:8]: No need to bump since those are all build dependencies

Comment (chenrui333) [reply Formula/guile.rb]: ➕

**Review by SMillerDev — APPROVED** [review]
Thanks @aconchillo  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by chenrui333** [comment]
networking failure for visp on 11
test timeout for dependency-check on linux build

good to be merged

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**